CHANGELOG.md

```diff
@@ -5,6 +5,121 @@ information on the list of deprecated flags and APIs please have a look at
+- Fix multiple `COPY --from` in multistage builds [moby/moby#34063](https://github.com/moby/moby/pull/34063)
++ Add path filtering to build session client [moby/moby#33859](https://github.com/moby/moby/pull/33859)
+- Fix handling of remote "git@" notation [moby/moby#33696](https://github.com/moby/moby/pull/33696)
+* Incrementally sending build context [docker/cli#231](https://github.com/docker/cli/pull/231)
+* Warn on empty continuation lines [moby/moby#33719](https://github.com/moby/moby/pull/33719)
+* Implement long running interactive session and sending build context incrementally [moby/moby#32677](https://github.com/moby/moby/pull/32677)
+- Fix copy —from conflict with force pull [moby/moby#33735](https://github.com/moby/moby/pull/33735)
+* Move file coping from the daemon to the builder [moby/moby#33454](https://github.com/moby/moby/pull/33454)
+- Fix behavior of absolute paths in .dockerignore [moby/moby#32088](https://github.com/moby/moby/pull/32088)
+- Fix Cache with ONBUILD [moby/moby#33524](https://github.com/moby/moby/pull/33524) [moby/moby#33500](https://github.com/moby/moby/pull/33500)
+* Refactor builder probe cache and container backend [moby/moby#33061](https://github.com/moby/moby/pull/33061)
+
+### Distribution
+
+* Stop trying to load images on an incompatible OS [moby/moby#33647](https://github.com/moby/moby/pull/33647)
+* Do not add duplicate platform information to service spec [moby/moby#33867](https://github.com/moby/moby/pull/33867)
+
+### Logging
+
+- Fix log readers can block writes indefinitely [moby/moby#33897](https://github.com/moby/moby/pull/33897)
++ Add stream format details for attach/logs endpoint [moby/moby#33494](https://github.com/moby/moby/pull/33494)
+- Fix wrong filemode for rotate log files [moby/moby#33926](https://github.com/moby/moby/pull/33926)
+* Expose config credentials without needing the Cli [docker/cli#227](https://github.com/docker/cli/pull/227)
+* System prune: only warn for volumes if --volumes is given [docker/cli#255](https://github.com/docker/cli/pull/255)
+- Fix stderr logging for journald and syslog [moby/moby#33832](https://github.com/moby/moby/pull/33832)
+* Pass the same log filename to logging plugins on start/stop requests [moby/moby#33583](https://github.com/moby/moby/pull/33583)
+
+### Networking
+
+* Include Endpoint List for Shared Endpoints [moby/moby#33662](https://github.com/moby/moby/pull/33662)
+* LCOW: owner and network endpoints [moby/moby#33774](https://github.com/moby/moby/pull/33774)
+* Host and Bridge network support in docker stack deploy [docker/cli#132](https://github.com/docker/cli/pull/132)
+
+### Runtime
+
+* Keep pause state when restoring container's status [moby/moby#34071](https://github.com/moby/moby/pull/34071)
+* Plugable secret backend [moby/moby#34157](https://github.com/moby/moby/pull/34157) [moby/moby#34123](https://github.com/moby/moby/pull/34123)
```

Comment:
Swarm mode; this is a feature - use external secret stores (!)

- CLI is not merged yet btw